modules/redpanda/redpanda_test.go

```diff
@@ -698,3 +705,23 @@ func TestRedpandaBootstrapConfig(t *testing.T) {
 		require.False(t, needsRestart)
 	}
 }
+
+func detectContainerHostAddress(image string, ctx context.Context) (string, error) {
+	c, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: testcontainers.ContainerRequest{
+			Image: image,
+		},
+		Started: false,
+	})
```

Comment:
suggestion: use [testcontainers.CleanupContainer](https://pkg.go.dev/github.com/testcontainers/testcontainers-go#CleanupContainer)

Reply:
IMHO, it is better to remove this (not started, temporary) container immediately (within this function) without waiting for completion of test (if I understand [testcontainers.CleanupContainer](https://pkg.go.dev/github.com/testcontainers/testcontainers-go#CleanupContainer) correctly).

Reply:
Done (it makes the test simpler).